Gemfile

```diff
@@ -12,6 +12,7 @@ end
 
 group :test do
   gem 'voxpupuli-test', '>= 1.0.0',  :require => false
+  gem 'puppet-lint-strict_indent-check', :require => false
```

Comment:
I didn't adjust the indent here because it will be purged by the next modulesync anyways.